integration-docs: Update to the new format (PagerDuty, Papertrail, Patreon, Pingdom). (#31233)
zulip/zulip: zerver/webhooks/pingdom/doc.md

# Zulip Pingdom integration

Zulip supports integration with Pingdom and can notify you of
uptime status changes from your Pingdom dashboard.

{start_tabs}

1. {!create-channel.md!}

1. {!create-an-incoming-webhook.md!}

1. {!generate-webhook-url-basic.md!}

> 1. In the main menu, go to **Integrations** under the **Integrations**

Reviewer: Good improvement indeed.

```suggestion
1. From the main menu, open the "Integrations" menu, and
```
To make the order of the instructions same as the traversal steps, as well as conformity.

1. Next, go to the following URL:
1. In the main menu, go to **Integrations** under the **Integrations**
    menu, and click on **Add integration**.

    `https://my.pingdom.com/reports/integration/settings`

> 

Author: Replaced the link which was confusing with text for clearity.

> 1. Configure the integration as follows and click **Save integration**:

Reviewer: I think it might be better to verbalize these settings, and remove the image.
For example: people who use screen readers might find it helpful this way.

> 1. Ensure your integration is selected in the **Check** settings.

Author: Referred the docs here: [doc1](https://support.atlassian.com/opsgenie/docs/integrate-opsgenie-with-pingdom/), [doc2](https://www.pagerduty.com/docs/guides/pingdom-integration-guide/)
